CHANGES.md

```diff
@@ -38,6 +38,9 @@ in family names to be split, e.g. `<foo>FAM<bar>`.
 directory during file parsing, to give the template processor access to
 workflow files.
 
+[#5571](https://github.com/cylc/cylc-flow/pull/5571) - Make workflow `CYLC_`
+variables available to the template processor during parsing.
+
```

Comment:
Will need to be moved. If #5634 goes in then the sentence should be written to `changes.d/5571.feat.md`.

Reply:
moved